src/Route.php

```diff
@@ -177,6 +184,18 @@ public function param($key, $default, $validator, $description = '', $optional =
         return $this;
     }
 
+    /**
+     * Set middleware status
+     *
+     * @return bool
+     */
+    public function middleware($useMiddleware = true): self
```

Comment:
Be consistent with other methods here, use just `$this->middleware`

Reply:
done 👍